src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -155,7 +155,8 @@ const createFormValidator = celebrate({
 })
 
 const duplicateFormValidator = celebrate({
-  [Segments.BODY]: BaseJoi.object<DuplicateFormBodyDto>({
+  // uses CreateFormBodyDto as that is the shape of the data used in client's WorkspaceService
+  [Segments.BODY]: BaseJoi.object<CreateFormBodyDto>({
```

Comment:
as discussed, might be better to stick to DuplicateFormBodyDto :)

Reply:
[d1c79ab](https://github.com/opengovsg/FormSG/pull/6755/commits/d1c79ab88b863193a8b2fb75c0b7b398d353a024)

[3661799](https://github.com/opengovsg/FormSG/pull/6755/commits/3661799d5c1d7f9dd9dabc26ee2ca69ae934523d)

Have separated out the overwrite params (to be ingested by `admin-form.service.ts`) and the duplicate form body dto that is to be passed between FE and BE! Should be clearer now!